PRODUCTION_DEPENDENCIES.md

```diff
@@ -0,0 +1,37 @@
+# Production Dependencies
+
+  | Name | Version | License |
+  | ---- | ------- | ------- |
+  | [angular-slider/ngx-slider](https://github.com/angular-slider/ngx-slider) | 2.0.4 | [MIT](/home/runner/work/greenforce-frontend/greenforce-frontend/node_modules/@angular-slider/ngx-slider/LICENSE) |
```

Comment:
issue: don't think the license path would work locally (i.e. `[MIT](/home/runner/work/...`). Would it be possible to not include that path?

Reply:
Link has been removed since I couldn't find a way to link to the license at github as the link to the license file requires a branch and we can't know if the dependency is using main, master or whatever.